src/micromamba/constructor.cpp

```diff
@@ -82,7 +82,9 @@ construct(const fs::path& prefix, bool extract_conda_pkgs, bool extract_tarball)
         {
             if (is_package_file(entry.path().filename().string()))
             {
+                LOG_TRACE << "Extracting " << entry.path().filename() << std::endl;
```

Comment:
we shoudl keep cout here. it's good to see the progress.

Reply:
I didn't removed it, just added the same info in the logs

Reply:
But the miniforge / mambaforge users won't see anything or am I mistaken? Since the TRACE log is not going to be displayed.

Reply:
have a look at line 86, it's still printed in `cout`

Reply:
then i don't know why we need the trace log :)